examples/custom-animation/growing-sequence-flow/README.md

```diff
@@ -33,9 +32,5 @@ It looks like `<path d="..." fill="none" stroke="black" stroke-width="1.78" stro
 ````
 
 ```javascript
-    const animatedSequenceFlowElt = bpmnVisualization.bpmnElementsRegistry.getElementsByIds(['sequence_flow_id'])[0].htmlElement;
-    animatedSequenceFlowElt.classList.add('growing');
+    bpmnVisualization.bpmnElementsRegistry.addCssClasses(['flow_5'], 'growing');
```

Comment:
For the readme, I prefer to keep as id  **'sequence_flow_id'** for the sequence flow. 
The user of the lib doesn't know what is and where is the id 'flow_5'.